src/stores/QueryStore.ts

```diff
@@ -84,14 +85,7 @@ export default class QueryStore extends Store<QueryStoreState> {
     }
 
     reduce(state: QueryStoreState, action: Action): QueryStoreState {
-        if (action instanceof SetPoint) {
-            const newState: QueryStoreState = {
-                ...state,
-                queryPoints: QueryStore.replacePoint(state.queryPoints, action.point),
```

Comment:
Very minor, but it looks like you changed the order of the if/else blocks for `SetPoint` and `InvalidatePoint` here for no real reason? Without this change it would be a bit easier to see what changed in this PR.

Reply:
You are right. I changed the order so that ```SetPoint```, ```AddPoint```, ```SetParametersAtOnce``` are close to each other. I can put this back however.

Reply:
No no that makes sense then!